lib/include/openamp/rpmsg_virtio.h

```diff
@@ -145,6 +164,39 @@ int rpmsg_init_vdev(struct rpmsg_virtio_device *rvdev,
 		    struct metal_io_region *shm_io,
 		    struct rpmsg_virtio_shm_pool *shpool);
 
+/**
+ * rpmsg_init_vdev_with_config - initialize rpmsg virtio device with config
+ * Master side:
```

Comment:
In keeping with the effort to remove potentially objectionable terminology, I suggest replacing "Master" and "Slave" with "Driver" and "Device" as in PR #323.

Reply:
I'm changing Master and Slave here to Host and Remote. It seems this header should use RPMsg terminology (host/remote), not virtio (driver/device), because it operates on rpmsg instances.